modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java

```diff
@@ -432,6 +462,54 @@ private JSONArray _getViewURLsJSONArray(
 			).toArray());
 	}
 
+	private boolean _isSharingEnabled(
+		long classNameId, long classPK, ThemeDisplay themeDisplay) {
+
+		Company company = null;
+
+		try {
+			company = _companyLocalService.getCompany(
+				themeDisplay.getCompanyId());
+		}
+		catch (PortalException portalException) {
+			if (_log.isDebugEnabled()) {
+				_log.debug(portalException);
+			}
+		}
+
+		SharingConfiguration companySharingConfiguration =
+			_sharingConfigurationFactory.getCompanySharingConfiguration(
+				company);
+
+		if (!companySharingConfiguration.isEnabled()) {
+			return false;
+		}
+
+		SharingConfiguration sharingConfiguration =
```

Comment:
Maybe I am wrong, but I think taking a look to getGroupSharingConfiguration, that it checked getCompanySharingConfiguration too.

Also, maybe would be easy to find where all this logic is used and create an API to share this logic